spec/models/embedded_ansible_worker/runner_spec.rb

```diff
@@ -133,12 +139,48 @@
 
         it "sets the embedded ansible job data retention value when the setting changes" do
           allow(embedded_ansible_instance).to receive(:alive?).and_return(true)
-          allow(runner).to receive(:provider).and_return(provider)
```

Comment:
Why can this be removed here, but not in the other examples above it?

Reply:
It was part of my change to add the `after`....I can remove this change if I remove the `after`.

We only needed this line in the other examples because we set an expectation on the `provider` object in those examples... therefore, we needed to make sure the `runner.provider` returns this object and not the completely different ruby object returned from `first_or_initialize` from the "normal" `provider` method.

Reply:
Sounds good